src/absil/ilread.fs

```diff
@@ -3970,7 +3970,8 @@ let ClosePdbReader pdb =
 #endif
 
 let OpenILModuleReader infile opts = 
-    if IL.runningOnWindows then
+
+    if not runningOnMono then
```

Comment:
It's not a blocker, but just to note that ``runningOnMono`` is not the opposite of ``runningOnWindows`` - you can run Mono on Windows, for example (though very few people do these days).